frontend/glideinFrontendElement.py

```diff
@@ -833,7 +839,13 @@ def iterate_one(self):
             # Only advertise if there is a valid key for encryption
             if key_obj is not None:
                 # add callback credential generator if needed
-                if not self.credentials_plugin.get_credentials(credential_purpose=CredentialPurpose.CALLBACK):
+                callback_creds = self.credentials_plugin.get_credentials(credential_purpose=CredentialPurpose.CALLBACK)
+                has_callback_cred = False
+                for cred in callback_creds:
+                    if cred.subject.split("@")[0] == glidein_el["attrs"]["GLIDEIN_Site"]:
+                        has_callback_cred = True
+                        break
+                if not has_callback_cred:
```

Comment:
Could a generic, not per-site, callback credential be set in the configuration? What would be its subject? Without "@"?